packages/app/screens/group/tabs/MembersScreen.js

```diff
@@ -1,8 +1,27 @@
-import React from "react";
+import MemberCard from "@app/components/MemberCard/MemberCard";
 import { View } from "react-native";
+import React from "react";
+import { useGuildContext } from "@app/utils/GuildContext";
 
 function MembersScreen() {
-  return <View></View>;
+  const { guildMembers } = useGuildContext();
+
+  return (
+    <View>
+      {guildMembers.map((member) => (
+        <View key={member.userId}>
+          <MemberCard
+            name={
+              (member.firstName + " " + member.lastName).length <= 12
+                ? member.firstName + " " + member.lastName
+                : (member.firstName + " " + member.lastName).substring(0, 12) +
+                  "..."
+            }
+          />
```

Comment:
Just a minor thing to fix/add in. Icons don't appear for some of the members that have avatars in the database and just has the no-pfp-icon by default, just need to add in the avatar image from the guild context into the MemberCard and you're good to go!